ci/set-branch.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+
+BRANCH=""
+for i in "$@"
+do
+    case $i in
+        -r=* | --repo=* )
+            REPO="${i#*=}"
+        ;;
+        -b=* | --branch=* )
+            BRANCH="${i#*=}"
+        ;;
+        * )
+        # unknown option
+        if [[ -n "$i" ]]
+        then
+            echo -e "\e[35m\e[1mUnknown input argument '$i'. Check CI .yml file\e[0m"
+            exit 1
+        fi ;;
+    esac
+    shift
+done
+
+echo -e "\e[35m\e[1mREPO   = ${REPO}\e[0m"
+echo -e "\e[35m\e[1mBRANCH = ${BRANCH}\e[0m"
+
+echo -e "\e[35m\e[1mGetting default branch by GitHub API of: ${REPO}\e[0m"
+
+DEFAULT_BRANCH=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/"${REPO}" 2>/dev/null | jq ".default_branch" -r)
```

Comment:
Why not use the git repo itself to get the default branch? That would free us from using the Github API.